app/build.gradle.kts

```diff
@@ -135,8 +135,8 @@ dependencies {
         add("standardImplementation", dependencyNotation)
     }
 
-    // TODO: this should use a versioned aar instead of referencing the project BITAU-94
-    implementation(project(":bridge"))
+    // TODO: this should use a versioned aar instead of referencing a local aar BITAU-94
+    implementation(files("libs/bridge-0.1.0-release.aar"))
```

Comment:
I'm still wondering if we should slap `snapshot` on here manually to make clear this isn't the _released_ version of `0.1.0`.

Reply:
I'll defer to you!

Our definition of _released_ is pretty nebulous. If its built and included in the source of the Bitwarden app, isn't it _released_?

I'm happy either way.

Reply:
I went ahead and added SNAPSHOT:

a02f475ab415274a596aef47d42f17c8a78d6b5b